robomaker-sample-app-ci/src/aws-robomaker-sample-application-ci.ts

```diff
@@ -110,18 +111,22 @@ async function setup() {
   try {
     await exec.exec("sudo", ["apt-key", "adv", "--fetch-keys", "http://packages.osrfoundation.org/gazebo.key"]);
 
-    const aptPackages = [
+    let aptPackages = [
       "zip",
       "cmake",
       "lcov",
       "libgtest-dev",
-      "python-pip",
-      "python-rosinstall",
       "python3-colcon-common-extensions",
+      "python3-apt",
       "python3-pip",
```

Comment:
Is `python3-pip` needed for all ubuntu distro? If not, we could do `(UBUNTU_DISTRO == "focal") ? "python3-pip" : "python-pip"` here to make it more consistent and we don't need the `if` below

Reply:
the distros before roxy were installing both